src/Fantomas.Core/CodePrinter.fs

```diff
@@ -2414,12 +2414,16 @@ and genMultilineAnonRecordAlignBrackets (isStruct: bool) fields copyInfo =
 
     let genAnonRecord =
         match copyInfo with
-        | Some ci -> sepOpenAnonRecd +> copyExpr fieldsExpr ci +> sepNln +> sepCloseAnonRecdFixed
+        | Some ci ->
+            sepOpenAnonRecd
```

Comment:
Would you mind doing the `genTriviaFor` thing as well for the openingBrace?
This isn't really required to solve the issues at hand but seems like a good thing to tag along